src/helpers.ts

```diff
@@ -822,8 +873,8 @@ export function getHostDevSourceObject(dev) {
         const slot = dev.source.address.slot;
         const func = dev.source.address.func;
         // In pci devices vendor and product is optional
-        const vendor = dev.source.vendor && dev.source.vendor.id;
-        const product = dev.source.product && dev.source.product.id;
+        const vendor = dev.source.address.vendor && dev.source.address.vendor.id;
+        const product = dev.source.address.product && dev.source.address.product.id;
```

Comment:
Was it wrong before or was things changed to make types easier?

Reply:
This was wrong before.